en/advanced/gimbal_control.md

```diff
@@ -1,82 +1,64 @@
-# Gimbal Control Setup
+# Gimbal Configuration
```

Comment:
@julianoes Normally we'd have a parent gimbal page, that might include some overview and purchase links. This is more like the camera page adjacent - it is all about config.

Are there any gimbals we know work?